creating_packages/define_abi_compatibility.rst

```diff
@@ -411,7 +411,7 @@ Let's see some examples, corresponding to common scenarios:
   they are linked and if they cross the frontiers of the API, it might also lead to
   incompatibilities. If the public headers have changed, it would depend on what changes and how are
   they used in ``MyLib/1.0``. Adding new methods to the public headers will have no impact, but
-  changing the implementation of some functions that will be inlined when compiled from
+  changing the implementation of some functions that will be inline when compiled from
```

Comment:
I think that verb tense was correct: those functions "will be inlined". It means the compiler will put them in some state: "the functions will be removed", "the functions will be compiled", "the functions will be inlined"

Reply:
It was marked as a typo by my spell-checker. Nevermind, it is now back